test/WebJobs.Script.Tests.Shared/TestHelpers.cs

```diff
@@ -112,6 +112,19 @@ public static void ClearHostLogs()
             }
 
             DirectoryInfo directory = GetFunctionLogFileDirectory(functionName);
+
+            if (!directory.Exists)
+            {
+                if (throwOnNoLogs)
+                {
+                    throw new InvalidOperationException($"{directory.FullName} does not exist.");
+                }
+                else
```

Comment:
We can get rid of this else...

Reply:
I ended up refactoring this a little to make it more streamlined.